packages/razzle/config/razzleDevServer.js

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env node
+'use strict';
+
+const devServer = require('webpack-dev-server');
+const chalk = require('chalk');
+
+class razzleDevServer extends devServer {
+
+  constructor(compiler, options = {}, _log) {
+    const verbose = options.verbose || false;
+    delete options['verbose'];
+    super(compiler, options, _log);
+    this.verbose = verbose;
+  }
+
+  showStatus() {
+
+    console.log(chalk.green(`> Webpack dev server started on port ${this.options.port}`));
```

Comment:
maybe remove this line or if you prefer to don't remove it keep it

just put an if statement that doesn't show this message when we start this with `--verbose` ...